packages/angular-cli/commands/serve.ts

```diff
@@ -103,6 +104,12 @@ const ServeCommand = Command.extend({
       default: false,
       description: 'Enable hot module replacement',
     },
+    {
+      name: 'inline-client',
+      type: String,
+      aliases: ['ic'],
+      description: '(Defaults to "http://host:port")'
```

Comment:
`Websocket client URL for Webpack inline mode (Defaults to "http://host:port")` sounds reasonable